### PR TITLE
Update updates.md

### PR DESCRIPTION
**Pull request by sh783**
Updated cadence from 0 0 0 3 to 0 0 0 7, per the change that finished deploying this week. Also added clarity around device restart expectations.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/managed-desktop/service-description/updates.md [hunk 1]
```diff
@@ -46,18 +46,19 @@ There are parts of the service that you manage, like app deployment, where it mi
 ## How update deployment works:
 1. Microsoft Managed Desktop deploys a new feature or quality update according to the schedule specified in the following table.
 2. During deployment, Microsoft Managed Desktop monitors for signs of failure or disruption based on diagnostic data and the user support system. If any are detected, we immediately pause the deployment to all current and future groups.
-    - Example: if an issue is discovered while deploying a quality update to the First group, then update deployments to First, Fast, and Broad will all be paused until the issue is resolved.
+    - Example: if an issue is discovered while deploying a quality update to the First group, then update deployments to First, Fast, and Broad will all be paused until the issue is mitigated.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/service-description/updates.md]:
```suggestion
    - Example: If an issue is discovered while deploying a quality update to the First group, then update deployments to First, Fast, and Broad will all be paused until the issue is mitigated.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @sh783, kindly commit the additional minor suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@sh783 Please implement the suggested changes. Thanks

**Comment by sh783** [comment]
@yogkumgit suggested change committed.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by sh783** [comment]
@JohanFreelancer9 can we please merge this today? We're getting questions about this inconsistent documentation.

**Comment by JohanFreelancer9** [comment]
@yogkumgit

**Comment by yogkumgit** [comment]
@jaimeo Please merge this PR. Thanks.

**Comment by sh783** [comment]
Pretty sure that Jaime is still on vacation. Who is handling this in his absence?

**Pull request merged**